x-pack/plugins/spaces/public/plugin.test.ts

```diff
@@ -43,7 +43,12 @@ describe('Spaces plugin', () => {
 
       management.sections.section.kibana = mockSection;
 
-      const plugin = new SpacesPlugin(coreMock.createPluginInitializerContext());
+      const mockInitializerContext = coreMock.createPluginInitializerContext();
+
+      // @ts-expect-error buildFlavor marked as readonly
+      mockInitializerContext.env.packageInfo.buildFlavor = 'traditional';
```

Comment:
@kc13greiner who/where can i ping for this?

Reply:
@pgayvallet Heya! Could you help us figure out the best way forward here?

Reply:
I'm not sure why I used `serverless` instead of `traditional` as default value here: https://github.com/elastic/kibana/blob/ab486aff0506a93bb249288e126c80f2cf8b9a81/packages/core/plugins/core-plugins-browser-mocks/src/plugins_service.mock.ts#L46

but that sounds like a mistake..

Anyway, regarding how to move forward, I would just change the signature of this function

https://github.com/elastic/kibana/blob/ab486aff0506a93bb249288e126c80f2cf8b9a81/packages/core/plugins/core-plugins-browser-mocks/src/plugins_service.mock.ts#L30

To something like

```ts
const createPluginInitializerContextMock = (config: unknown = {}, { buildFlavor = 'serverless'}: { buildFlavor?: BuildFlavor} = {})
```

and then use it in the function accordingly.

Would that work with you?

Reply:
@pgayvallet That works! Thank you!